2017/ru/0x00-header.md

```diff
@@ -0,0 +1,19 @@
+![OWASP LOGO](images/OWASP_logo.png)
+
+## Топ-10 OWASP 2017
+
+Десять самых опасных угроз безопасности веб-приложений
+
+
+
+
+![WASP Logo URL TBA](images/front-wasp.png)
+
+|  | ![Creative Commons License Logo](images/front-cc.png) |
+| -- | -- |
+| https://owasp.org | Данная работа выпущена под лицензией Creative Commons Attribution-ShareAlike 4.0 International License |
```

Comment:
Format tables if possible here and below. I recommend using [Markdown Table Formatter](https://marketplace.visualstudio.com/items?itemName=josa.markdown-table-formatter) or another table extension.

Reply:
I'll try to do that for all tables.